managed/services/telemetry/config.default.yml

```diff
@@ -8,6 +8,22 @@ telemetry:
       - metric_name: "pmm_server_ia_enabled"
         column: "ia_enabled"
 
+  - id: PMMServerAlertsFire
+    source: VM
+    query: max_over_time(grafana_alerting_alerts{state="alerting"}[24h])
+    summary: "Max Number of Alerts Firing over last 24h"
+    data:
+      - metric_name: "pmm_server_alerts_firing_max_24h"
```

Comment:
```suggestion
      - metric_name: "pmm_server_max_active_concurrent_alerts_24h"
```

Reply:
I think metric name it better to leave as it is, it is not intended to be "correct English", but rather the same prefix for similar metrics.
e.g. pmm_server_alerts_firing_max_24h, pmm_server_alerts_firing_max_12h